### PR TITLE
Cleaned up README

### PR DESCRIPTION
**Pull request by jeffmay**
@DSASanFrancisco/portal-members 

Fixed the issue with having the wrong version of python.

Also, I moved the troubleshooting stuff to a separate section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,20 +79,8 @@ to use the site.
...
-        ```
-    2. (NOTE) If you can't connect to the mysql socket `/tmp/mysql.sock`, you need to start the server
-        ```
-        mysql.server start
-        ```
-    3. (NOTE) If you see an error about a missing the `dsa` database, create it with
-        ```
-        mysql -u root -e "create database dsa"
-        ```
+    alembic upgrade head
```
Comment (jessebutterfield) [inline README.md]: Do we not still need 'PYTHONPATH=. alembic upgrade head'?

Comment (jeffmay) [inline README.md]: Apparently not? @stevengoldberg ran the command without it and it worked.

Comment (jeffmay) [inline README.md]: I think the way that `python -m venv .` works is effectively that it sets the `pythonpath=.` and creates a `./bin` and `./lib` so maybe that's why.

Comment (jessebutterfield) [inline README.md]: Cool. I guess if people have problems we can add it to the troubleshooting.

**Pull request merged**